config/routes.rb

```diff
@@ -5,6 +5,10 @@
     get "/about", to:"static_pages#about"
     get "/contact", to:"static_pages#contact"
     get "/signup", to: "users#new"
-    resources :users
+    post "/signup", to: "users#create"
+    get "/login", to: "sessions#new"
+    get "/logout", to: "sessions#destroy"
+    post "/login", to: "sessions#create"
+    resources :users, only: %i(new create show)
```

Comment:
dùng câu lệnh gì để xem toàn bộ url routes của hệ thống e nhỉ

Reply:
dạ dùng lệnh rails routes ạ